From e547e35f44ecf2d1f2e05a3c9c5f77c76de08e50 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Mon, 24 Jul 2023 17:51:04 +0200
Subject: [PATCH 0415/1141] arm64: defconfig: add CONFIG_OF_OVERLAY in addons
 fragment

Enable CONFIG_OF_OVERLAY to have dt symbols and overlay available.

Change-Id: If437fd669cf243c65b65599dd133c686566b458d
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 672498f89b8a..67c49b626bd2 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -50,6 +50,7 @@ CONFIG_NET_SCH_MULTIQ=m
 CONFIG_NET_SCH_PRIO=m
 CONFIG_NET_SCH_SKBPRIO=m
 CONFIG_NVMEM_STM32_ROMEM=y
+CONFIG_OF_OVERLAY=y
 CONFIG_PCIE_STM32=y
 CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
-- 
2.39.2

